pkg/operator/ceph/object/spec.go

```diff
@@ -58,6 +58,10 @@ func (c *clusterConfig) createDeployment(rgwConfig *rgwConfig) (*apps.Deployment
 		return nil, err
 	}
 	replicas := int32(1)
+	// On Pacific, we can use the same keyring and have dedicated rgw instances reflected in the service map
+	if c.clusterInfo.CephVersion.IsAtLeastPacific() {
```

Comment:
Do we automatically add preferred pod anti-affinity so the rgw services don't end up on the same node? It's a separate question from this PR, but worth following up

Reply:
I thought about that but just remembered that this was at the pod level. I didn't dig too much assuming it will continue to work. Let me look deeper but we can address this in another PR for sure.

Reply:
I guess the answer is that we do not, just like CephFS we have a section in the YAML to enforce it. Let me add the same to the example.